### PR TITLE
Adding initial responses + easter eggs

### PR DESCRIPTION
**Pull request by seriouslysean**

### CODE REVIEW

File: skills/easter-eggs.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * When the bot hears a specific phrase, show an easter egg
+ */
+
+module.exports = (controller) => {
+
+    // When someone asks what does the fox say, show a gif!
+
+    const whatDoesTheFoxSayString = [
```
Comment (seriouslysean) [inline skills/easter-eggs.js:9]: Listen to any mention for `what does the fox say?` with or without the question mark.

**Review by dgautsch — APPROVED** [review]

File: skills/easter-eggs.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        'what does the fox say\\??',
+    ];
+
+    controller.hears(whatDoesTheFoxSayString, 'direct_mention, ambient, mention', (bot, message) => {
+        console.log('heard something!', message);
+
+        bot.reply(message, {
+            attachments: [{
+                text: `What does the fox say?!`,
+                image_url: 'https://media.giphy.com/media/jPRBtKha7rO6I/giphy.gif',
```
Comment (seriouslysean) [inline skills/easter-eggs.js:20]: In order to show an image, you need to have some text in the message response object.

File: skills/messaging-help.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
+/**
+ * When the bot joins a channel, tell everyone what it can do
+ */
+const config = require('../config');
+
+const helpMessage = `I'm here to help you find out what PRs you currently have open and which of them need your attention. In order to get the most out of me, you can use the following slash commands:
```
Comment (seriouslysean) [inline skills/messaging-help.js:6]: Multi-line string literals are the best way to format messages from what I've found so far.

File: skills/messaging-help.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+* \`/${config.UGB_SLASH_COMMAND} pr reviewer\` - Pull PRs where you are a reviewer
+* \`/${config.UGB_SLASH_COMMAND} pr assigned\` - Pull PRs where you are an assignee
+* \`/${config.UGB_SLASH_COMMAND} help\` - Additional information
+* \`/${config.UGB_SLASH_COMMAND} creators\` - The lovely folks who made me`;
+
+module.exports = (controller) => {
+
+    // Bot joins a channel
+
+    controller.on('bot_channel_join,bot_group_join', (bot, message) => {
```
Comment (seriouslysean) [inline skills/messaging-help.js:18]: Whenever the bot joins a channel or a group, send the help reply.

File: skills/messaging-help.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+        bot.reply(message, {
+            attachments: [{
+                title: `Beep, boop! I'm the ${config.UGB_BOT_NAME}!`,
+                pretext: `Thanks for inviting me, <@${message.inviter}>!`,
+                text: helpMessage,
+                mrkdwn_in: [
```
Comment (seriouslysean) [inline skills/messaging-help.js:25]: Enable markdown support in the pretext and text.

File: skills/messaging-help.js [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+        const messageObj = {
+            attachments: [{
+                pretext: `Don't worry-- *${config.UGB_BOT_NAME}* to the rescue, <@${message.user}>!`,
+                text,
+                mrkdwn_in: [
+                    'text',
+                    'pretext',
+                ],
+                ...(showEasterEgg ? {
```
Comment (seriouslysean) [inline skills/messaging-help.js:62]: 1 in 5 chance to show the easter egg 😂

File: skills/messaging-help.js [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+                    image_url: 'https://media.giphy.com/media/F7yLXA5fJ5sLC/giphy.gif',
+                } : {}),
+            }],
+        };
+
+        bot.reply(message, messageObj);
+    });
+
+    // Someone asks who created me
```
Comment (seriouslysean) [inline skills/messaging-help.js:71]: Shout out to us! 💪

File: skills/messaging-help.js [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+    const creatorStrings = [
+        '^who made you\??$',
+        '^creators$',
+    ];
+
+    controller.hears(creatorStrings, 'direct_mention', (bot, message) => {
+        bot.reply(message, {
+            attachments: [{
+                pretext: `Thanks for asking, <@${message.user}>!`,
```
Comment (seriouslysean) [inline skills/messaging-help.js:81]: Replies with an @ mention to the user who asked.

File: skills/messaging-help.js [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+module.exports = (controller) => {
+
+    // Bot joins a channel
+
+    controller.on('bot_channel_join,bot_group_join', (bot, message) => {
+
+        bot.reply(message, {
+            attachments: [{
+                title: `Beep, boop! I'm the ${config.UGB_BOT_NAME}!`,
+                pretext: `Thanks for inviting me, <@${message.inviter}>!`,
```
Comment (seriouslysean) [inline skills/messaging-help.js:23]: Respond to the person who invited the bot.

**Pull request merged**